src/lsp/cobol_ast/misc_descr.ml

```diff
@@ -721,21 +725,46 @@ let pp_arithmetic_mode ppf = function
   | StandardDecimal -> Fmt.pf ppf "STANDARD-DECIMAL"
 
 let pp_options_clause ppf = function
-  | Arithmetic am -> Fmt.pf ppf "ARITHMETIC %a" pp_arithmetic_mode am
+  | Arithmetic am ->
+      Fmt.pf ppf "ARITHMETIC %a" pp_arithmetic_mode am
   | DefaultRoundedMode rm ->
-    Fmt.pf ppf "DEFAULT ROUNDED MODE %a" pp_rounding_mode rm
+      Fmt.pf ppf "DEFAULT ROUNDED MODE %a" pp_rounding_mode rm
   | EntryConvention ec ->
-    Fmt.pf ppf "ENTRY-CONVENTION %a" pp_entry_convention ec
+      Fmt.pf ppf "ENTRY-CONVENTION %a" pp_entry_convention ec
   | FloatBinaryDefault em ->
-    Fmt.pf ppf "FLOAT-BINARY %a" Data_descr.pp_endianness_mode em
+      Fmt.pf ppf "FLOAT-BINARY %a" Data_descr.pp_endianness_mode em
   | FloatDecimalDefault ee ->
-    Fmt.pf ppf "FLOAT-DECIMAL %a" Data_descr.pp_encoding_endianness ee
+      Fmt.pf ppf "FLOAT-DECIMAL %a" Data_descr.pp_encoding_endianness ee
   | IntermediateRounding rm ->
-    Fmt.pf ppf "INTERMEDIATE-ROUNDING %a" pp_rounding_mode rm
+      Fmt.pf ppf "INTERMEDIATE-ROUNDING %a" pp_rounding_mode rm
 
 let pp_options_paragraph : options_paragraph Fmt.t =
-  Fmt.(
-    any "OPTIONS.@ " ++
-    box (list ~sep:sp (pp_with_loc pp_options_clause))
-    ++ any "."
-  )
+  Fmt.(any "OPTIONS.@ " ++
+       box (list ~sep:sp (pp_with_loc pp_options_clause)) ++
+       any ".")
+
+
+(* --- *)
+
+let pp_informational_paragraph_header ppf = function
+  | Author -> Fmt.pf ppf "AUTHOR"
+  | DateCompiled -> Fmt.pf ppf "DATE-COMPILED"
+  | DateModified -> Fmt.pf ppf "DATE-MODIFIED"
+  | DateWritten -> Fmt.pf ppf "DATE-WRITTEN"
+  | Installation -> Fmt.pf ppf "INSTALLATION"
+  | Remarks -> Fmt.pf ppf "REMARKS"
+  | Security -> Fmt.pf ppf "SECURITY"
+
+let pp_comment_entry: comment_entry Pretty.printer =
+  Fmt.(list ~sep:sp string)
+
+let pp_informational_paragraph: informational_paragraph Pretty.printer =
+  Fmt.(any "@[<4>" ++                       (* <- indent by 4 to avoid Area A *)
```

Comment:
Weird that there is not a construct to do that, like symbolically, ie create a box with an indentation with automatic close at the end. All these strings as format descriptions are heaven for bugs in formatting.

Reply:
What you describe is `Fmt.box ~indent:4`

Reply:
Ah yes, good idea ;-)